apycula/clock_fuzzer.py

```diff
@@ -266,9 +266,23 @@ def spine_aliases(quads, dests, clks):
                     aliases[spine_row, tap, dest] = ct[0], ct[1], dest
     return aliases
 
+# add border cells
+def add_rim(rows, cols):
+    if 1 in rows:
```

Comment:
so does this only deal with the left rim? What about the `db.rows` column/row? I'd assume we need to add those to the next one as well.

Reply:
they are added via else: clauses